src/Draco.Compiler/Api/Syntax/SyntaxFactory.cs

```diff
@@ -119,18 +142,32 @@ public static SeparatedSyntaxList<TNode> SeparatedSyntaxList<TNode>(SyntaxToken
     public static VariableDeclarationSyntax VariableDeclaration(
         string name,
         TypeSyntax? type = null,
-        ExpressionSyntax? value = null) => VariableDeclaration(true, name, type, value);
+        ExpressionSyntax? value = null) => VariableDeclaration(null, true, name, type, value);
+
+    public static VariableDeclarationSyntax VariableDeclaration(
+        SyntaxToken? visibility,
```

Comment:
I'd still suggest changing this overload to use Visibility instead of a token, as this is a higher level API (using strings instead of tokens, ...) and you can just use VisibilityToken for impl